src/Cms/Responder.php

```diff
@@ -70,6 +79,50 @@ public function body(string $body = null)
         return $this;
     }
 
+    /**
+     * Setter and getter for the cache expiry
+     * timestamp for Kirby's cache
+     *
+     * @param int|string|null $cacheExpiry Timestamp or time string to parse
+     * @param bool $override If `true`, the already defined timestamp will be overridden
+     * @return int|null|self
+     */
+    public function cacheExpiry($cacheExpiry = null, bool $override = false)
```

Comment:
Should we call the method and prop rather just `expires`? I feel that would be more in style with many other Kirby methods.

Reply:
All other methods in this class are nouns, so `expires` wouldn't be in line with them as the only verb. I'm split on this though, I think we need more opinions.

Reply:
I'd vote for `$kirby->response()->expires()` as well. To me it feels closer to a command like `$kirby->response()->redirect()`